tools/local-snapshot-runner-app/src/fakeUrlResolver.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRequest } from "@fluidframework/core-interfaces";
+import { IContainerPackageInfo, IResolvedUrl, IUrlResolver, } from "@fluidframework/driver-definitions";
+import { IOdspResolvedUrl } from "@fluidframework/odsp-driver-definitions";
+
+/**
+ * Fake URL resolver that returns hard coded values on every request
+ */
+export class FakeUrlResolver implements IUrlResolver {
+
+    public async resolve(request: IRequest): Promise<IResolvedUrl | undefined> {
+        const fakeOdspResolvedUrl: IOdspResolvedUrl = {
+            type: "fluid",
+            odspResolvedUrl: true,
+            id: "1",
+            siteUrl: request.url,
+            driveId: "1",
+            itemId: "1",
+            url: request.url,
+            hashedDocumentId: "1",
+            endpoints: {
+                snapshotStorageUrl: request.url,
```

Comment:
Can we put undefined for URIs (even it TS time does not allow it)? I'd rather see code crashing on undefined then do network call (assuming we made a mistake and got to a place that is about to try to reach out to network). It feels like it would be easier to debug (having callstack)

Reply:
Do you mean to use something like `undefined!`?

Reply:
undefined is good if code allows it (does not assert it has to be defined) - that seems like the best outcome.
If we have to use some values (for example, driverId & itemId will be used to generate docId that is added to all telemetry), then I'd prefer to see some unique value that is easy to find in code base. Guid will do it, but they are too long and carry less context (without search repo). "LocalOdspDriver" seems like reasonable string to use (or something like that)

Reply:
`IOdspResolvedUrl` requires it to be defined. Would something like `FluidRunner-FakeUrlResolver` work? Or should I reuse the guid used for all the other string properties?

Reply:
I'd just use "FakeUrlResolver". It's sufficient, and code can move (over time) and be reused, so I'd not couple it to particular usage (FluidRunner)